### PR TITLE
UPDATE: Add data-test to all components

### PR DESCRIPTION
**Pull request by wEprous**
`data-test` is needed for Cypress tests

### CODE REVIEW

**Comment by wEprous** [comment]
@tomashapl should be the global flow type placed in folder `globals` or `common`? I will have some common/global stuff in `Stack`

File: .github/contribution/component-design.md [hunk 1]
```diff
@@ -50,13 +50,15 @@ const Component = (props: Props) => {
 ## Component Flow typing
 For static typing we use [Flow](https://flow.org/en/docs/react/). Always create `index.js.flow` file, where the declaration of types is stored. We use `strict` and `read-only` types to be more strict in setting up values of the props, `optional` types are used only when the purpose of the prop is really optional.
 
+Every component should have `dataTest` prop. For this cases, we are using `globals` flow type.
 ```jsx
 // @flow
-import defaultTokens from "../defaultTokens";
+import type { globals } from "../common/common.js.flow";
```
Comment (tomashapl) [inline .github/contribution/component-design.md]: Please types should begin with Uppercase letter 😉

Comment (wEprous) [reply .github/contribution/component-design.md]: So, approved or not? 🤔 😂 I can edit that quickly

Comment (tomashapl) [reply .github/contribution/component-design.md]: Sorry miss click 🙈, can you please update this ?

Comment (wEprous) [reply .github/contribution/component-design.md]: Working on it, it'll be nice to release it today.

**Pull request merged**